dagger-common/src/main/java/io/odpf/dagger/common/serde/DaggerInternalTypeInformation.java

```diff
@@ -0,0 +1,26 @@
+package io.odpf.dagger.common.serde;
```

Comment:
shall we move this inside deserialisation?

Reply:
Makes sense actually.